target/linux/ramips/dts/PSG1218.dts

```diff
@@ -0,0 +1,121 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+/ {
+	compatible = "PSG1218", "ralink,mt7620a-soc";
+	model = "Phicomm PSG1218";
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		red {
+			label = "phicomm:red:status";
+			gpios = <&gpio0 8 1>;
+		};
```

Comment:
The naming of **all** LEDs is wrong. It should be something like:

```
<function> {
  label = "<boardname>:<colour>:<function>";
  gpios = <&gpio0 8 1>;
};
```

Have a look at the PSG1208.dts file to see how how should be done.